### ISSUE TITLE
please support custom current promisePolyfillJS

### ISSUE THREAD

**Issue by virskor** [pichillilorenzo/flutter_inappwebview#302]
please support custom promisePolyfillJS.
 this is good plugin, but i found my webpage dose not work well with this plugin, after debugging i found const promisePolyfillJS on IOS side would confuse us.

not everyone use same polyfill version, please support this feature.
THIS IS AN IMPORTANT ISSUE.
THANKS FOR YOUR GREAT JOB.

**Comment by virskor** [comment]
/Classes/InAppWebView.swift

line about 43
```js
// // https://github.com/taylorhakes/promise-polyfill/blob/master/src/index.js
// let promisePolyfillJS = """
// if (window.Promise == null) {
//   var setTimeoutFunc = setTimeout;
//   function isArray(x) {
//     return Boolean(x && typeof x.length !== "undefined");
//   };
//   function noop() {}
//   function bind(fn, thisArg) {
//     return function() {
//       fn.apply(thisArg, arguments);
//     };
//   };
//   function Promise(fn) {
```

**Comment by virskor** [comment]
if anyone has same problem with me, just comment it!!

**Comment by pichillilorenzo** [comment]
Released new version `3.0.0` with updated polyfill: https://github.com/tildeio/rsvp.js

Let me know if your code works now.. However this polyfill is added only in webviews that are too old and where window.Promise is not defined.. I use it because I need it to make javascript bridge work properly!

**Comment by virskor** [comment]
It worked fine now
